### PR TITLE
[KEYCLOAK-1431] Update page titles

### PR DESCRIPTION
**Pull request by girirajsharma**

### CODE REVIEW

File: forms/common-themes/src/main/resources/theme/base/admin/resources/partials/brute-force.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-9 col-md-10 col-sm-push-3 col-md-push-2">
-    <h1><strong>Settings</strong> {{realm.realm|capitalize}}</h1>
```
Comment (stianst) [inline forms/common-themes/src/main/resources/theme/base/admin/resources/partials/brute-force.html]: We shouldn't use <strong> in the page title

File: forms/common-themes/src/main/resources/theme/base/admin/resources/partials/client-role-detail.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
     </ol>
 
     <h1 data-ng-show="create"><strong>Add Role</strong></h1>
-    <h1 data-ng-hide="create"><strong>Role</strong> {{role.name}}</h1>
```
Comment (stianst) [inline forms/common-themes/src/main/resources/theme/base/admin/resources/partials/client-role-detail.html]: Add '|capitalize' to this to make it consistent with 'client.clientId'

File: forms/common-themes/src/main/resources/theme/base/admin/resources/partials/federated-generic.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
         <li data-ng-show="create">Add User Federation Provider</li>
     </ol>
 
-    <h1 data-ng-hide="create"><strong>{{instance.providerName|capitalize}} User Federation Provider</strong> {{instance.displayName|capitalize}}</h1>
+    <h1 data-ng-hide="create"><strong>{{instance.providerName|capitalize}} User Federation Provider</strong></h1>
```
Comment (stianst) [inline forms/common-themes/src/main/resources/theme/base/admin/resources/partials/federated-generic.html]: Remove 'User Federation Provider'. For clients it's just clientId, so should be consistent.

File: forms/common-themes/src/main/resources/theme/base/admin/resources/partials/role-detail.html [hunk 4]
```diff
@@ -6,7 +6,7 @@
         <li data-ng-show="create">Add Role</li>
     </ol>
 
-    <h1 data-ng-hide="create"><strong>Role</strong> {{role.name}}</h1>
+    <h1 data-ng-hide="create">{{role.name}}</h1>
```
Comment (stianst) [inline forms/common-themes/src/main/resources/theme/base/admin/resources/partials/role-detail.html]: |capitalize

**Comment by girirajsharma** [comment]
Fixed inconsistencies.

**Pull request merged**